libnavigation-util/src/test/java/com/mapbox/navigation/utils/internal/LoggerProviderTest.kt

```diff
@@ -1,73 +1,58 @@
 package com.mapbox.navigation.utils.internal
 
-import com.mapbox.base.common.logger.Logger
-import io.mockk.every
-import io.mockk.mockk
-import io.mockk.mockkObject
-import io.mockk.unmockkObject
+import com.mapbox.common.Logger
+import com.mapbox.navigation.testing.MockLoggerRule
 import io.mockk.verify
-import org.junit.After
-import org.junit.Before
+import org.junit.Rule
 import org.junit.Test
 
 class LoggerProviderTest {
 
-    private lateinit var mockkLogger: Logger
-
-    @Before
-    fun setup() {
-        mockkLogger = mockk(relaxUnitFun = true)
-        mockkObject(LoggerProvider)
-        every { LoggerProvider.logger } returns mockkLogger
-    }
-
-    @After
-    fun reset() {
-        unmockkObject(LoggerProvider)
-    }
+    @get:Rule
+    val mockLoggerTestRule = MockLoggerRule()
 
     @Test
     fun `alias log V`() {
```

Comment:
nit, to keep documented
```suggestion
    fun `alias log V (called lvl D, Logger doesn't provide V)`() {
```